File: Resource/QLNH-Freelance/ComNha_API/ComNha_API/Models/Khachhang.cs
using System;
using System.Collections.Generic;

namespace ComNha_API.Models
{
    public partial class Khachhang
    {
        public int MaKhachHang { get; set; }
        public string SoDienThoai { get; set; } = null!;
        public string TenKhachHang { get; set; } = null!;
        public string? DiaChi { get; set; }
        public string? Email { get; set; }
    }
}

File: Dev/Models/Models/KhachHang.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name="khachhang")
public class KhachHang {
    @Id
    @GeneratedValue
    private Long MaKhachHang;
    private String SoDienThoai;
    private String TenKhachHang;
    private String DiaChi;
    private String Email;
}